### PR TITLE
feat(Titles): add new Title2, replace Title2 and Title3 with Title3 and Title4 respectively

### PR DESCRIPTION
**Pull request by marcoskolodny**
BREAKING CHANGE: renamed `Title2` to `Title3` and `Title3` to `Title4`.

Issue: [Link](https://jira.tid.es/browse/WEB-1982)

### CODE REVIEW

File: playroom/snippets.tsx
Comment (yceballost) [inline playroom/snippets.tsx:1]: "Title4 is not defined" in playroom. Missing to import in playroom maybe?

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__stories__/title-story.tsx [hunk 1]
```diff
@@ -89,6 +89,18 @@ export const Title3Story: StoryComponent<Args> = ({title, right, linkText}) => (
     />
 );
 
+export const Title4Story: StoryComponent<Args> = ({title, right, linkText}) => (
+    <TitleComponent
+        as="h4"
```
Comment (atabel) [inline src/__stories__/title-story.tsx:94]: I think these `as` are wrong, bigger tittle (Title4) should have the h1

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

File: src/__stories__/title-story.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/__stories__/title-story.tsx]: Good catch. Fixed

**Pull request merged**